js/main.js

```diff
@@ -58,4 +58,43 @@ window.onload = () => {
   }
 
   Book.displayBooks();
-};
+  const listLink = document.querySelector('#list-link');
+  const newLink = document.querySelector('#new-link');
+  const contactLink = document.querySelector('#contact-link');
+
+  const listSection = document.querySelector('.books-list');
+  const newSection = document.querySelector('.add-books');
+  const contactSection = document.querySelector('.contact');
+
+  listLink.addEventListener('click', () => {
+    listSection.classList.add('show');
+    newSection.classList.remove('show');
+    contactSection.classList.remove('show');
+    listLink.classList.add('active');
+    newLink.classList.remove('active');
```

Comment:
[OPTIONAL]

- I would recommend clearing the input field whenever we add one book and also add restrictions so that the user can't enter an empty book.

Reply:
Done